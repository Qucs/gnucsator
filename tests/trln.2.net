# Qucs

#Pac:P1 _net0 gnd Num="1" Z="50 Ohm" P="0" f="1 GHz" Temp="26.85"

Pac:P2 _net0 gnd Num="1" Z="50 Ohm" P="1" f="1 GHz" Temp="26.85"
TLIN:Line1 _net0 _net1 Z="50 Ohm" L="100 mm" Alpha="0 dB" Temp="26.85"
TLIN:Line2 _net1 _net2 Z="50 Ohm" L="100 mm" Alpha="0 dB" Temp="26.85"

R:R1 gnd _net2 R="100 Ohm"
R:R2 gnd _net1 R="100 Ohm"

.AC:SP1 Type="log" Start="1e5" Stop="1e9" Points="6"
