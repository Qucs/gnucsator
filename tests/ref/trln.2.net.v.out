
module main ();
//# Qucs 
Pac #(.Num(1),.P(1),.Temp(26.85),.Z(50),.f(1G)) P2 (.1(_net0),.2(0));
TLIN #(.Alpha(0),.L(100m),.Temp(26.85),.Z(50)) Line1 (.a(_net0),.b(_net1));
TLIN #(.Alpha(0),.L(100m),.Temp(26.85),.Z(50)) Line2 (.a(_net1),.b(_net2));
R #(.R(100)) R1 (.p(0),.n(_net2));
R #(.R(100)) R2 (.p(0),.n(_net1));
endmodule // main

